### PR TITLE
refractor(helm-docs task):to make it execute only once

### PR DESCRIPTION
**Pull request by Pavan-Nambi**
issue:
https://github.com/apache/apisix-helm-chart/issues/438#issue-1517120651

### CODE REVIEW

**Review by tao12345666333 — CHANGES_REQUESTED** [review]
You need to follow the format of GitHub action files, thanks

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,5 +1,17 @@
 name: CI
 
+name: Run helm-docs
+        run: |
```
Comment (tao12345666333) [inline .github/workflows/ci.yaml:4]: The indentation of it is incorrect.

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -9,15 +9,40 @@ on:
 
 jobs:
 
+  helm_docs:
+    name: Generate Helm Docs
```
Comment (tao12345666333) [inline .github/workflows/ci.yaml:13]:
```suggestion
    name: Check the Helm Docs format
```

**Review by tao12345666333 — APPROVED** [review]
LGTM

Thanks!

**Review by Gallardot — APPROVED** [review]

**Pull request merged**